etc/inc/filter.inc

```diff
@@ -11,7 +11,7 @@
 	originally part of m0n0wall (http://m0n0.ch/wall)
 	Copyright (C) 2003-2004 Manuel Kasper <mk@neon1.net>.
 	All rights reserved.
-
+ne
```

Comment:
Yeh, I guess I was pressing random keys at the time, and just kept going through to submit pull request without looking.
I see you put the correct fix in yourself - good.